app/routes.js

```diff
@@ -1,36 +1,52 @@
 var config = require('./config');
 var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn;
 var express = require('express');
-var middleware = require('./middleware');
+var authorisation = require('./authorisation');
```

Comment:
This should be spelled `authorization`. The concept of authorization in technology is abbreviated to "authz", and it is already spelled -ize in existing code in `middleware.js`.

Reply:
This always confused me as in American they use `z`s but in UK we use `s`es.

In the company were I used to work they preferred `s`es because "we're in UK innit?" - should we stick with the American spelling or the British one? (not that I mind either way)

I'll change to the `z` here then.